client/components/relatedWidget/ProductListItem.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import StarAverage from '../shared/StarAverage.jsx';
+
+class ProductListItem extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render () {
+    // console.log('image:', this.props.image);
+    return (<div
+      onClick={()=>{ this.props.getNewProduct(this.props.text.id); }}
+      className={`menu-item ${this.props.selected ? 'active' : ''}`}
+      >
+        <h4>{this.props.text.category}</h4>
+        <h4>{this.props.text.name}</h4>
+        <img src={this.props.image} width='200' height='250'/>
```

Comment:
Don't forget to add an 'alt' attribute for accessibility, could probably just be the product name.